code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -705,3 +710,44 @@ var/list/ai_verbs_default = list(
 
 /mob/living/silicon/ai/handle_reading_literacy(var/mob/user, var/text_content, var/skip_delays, var/ai_show)
 	. = ai_show ? ..(user, text_content, skip_delays) : stars(text_content)
+
+/mob/living/silicon/ai/proc/ai_take_image()
+	set name = "PHOTO: Make"
```

Comment:
I think verb names would normally be more like `Take Photo` rather than this format (and below).